services/generic/barrier_control.md

```diff
@@ -1,36 +1,94 @@
+| in   | cmd.notiftype.get_report | null       |              | Requests notification types the device is using while opening/closing door.                                                                                                |
+| in   | cmd.notiftype.set        | bool_map   |              | Sets notification type the device is is using while opening/closing door. Configurable notification types are defined in [`sup_notiftypes`](#service-properties) property. |
+| out  | evt.notiftype.report     | bool_map   |              | Returns notification types the device is using while opening/closing door.                                                                                                 |
+| -    |                          |            |              |                                                                                                                                                                            |
+| in   | cmd.op.stop              | null       |              | Commence emergency stop of any operation.                                                                                                                                  |
+| -    |                          |            |              |                                                                                                                                                                            |
+| in   | cmd.state.get_report     | null       |              | Requests the current state of the device.                                                                                                                                  |
+| out  | evt.state.report         | string     | `stopped_at` | Reports the current state of the device, one of values defined in [`sup_states`](#service-properties) property.                                                            |
+| -    |                          |            |              |                                                                                                                                                                            |
+| in   | cmd.tstate.set           | string     |              | Sets the target state of the device to the one of values defined in [`sup_tstates`](#service-properties) property.                                                         |
+
+## Interface properties
+
+| Name         | Required | Example | Description                                                                                                                    |
+|--------------|----------|---------|--------------------------------------------------------------------------------------------------------------------------------|
+| `stopped_at` | No       | `"30"`  | Position as percentage value at which the barrier stopped on emergency halt, where `1` is near closed while `99` is near open. |
```

Comment:
this shoudl be 'cur_pos' or sth similr and it should represent current position of the barrier, we need to control and report to see in the app if window shade is 30 % or what and such report chould be sent every ~5% to inform user about the progress

Reply:
Leaving discussion for post mortem.